### PR TITLE
OCL_12.1.12.1_Updates

### PR DESCRIPTION
**Pull request by ManjushaVaidya**
- Migrated OCL version 13.1.4.2 to 12.1.2.1 in Github Markdown and documented 12.1.2.1 updates.

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**
- [x] This PR adds new content

### CODE REVIEW

File: product_docs/docs/ocl_connector/12.1.2.1/index.mdx [hunk 1]
```diff
@@ -14,3 +14,22 @@ legacyRedirectsGenerated:
 ---
 
 <StubCards product='ocl_connector' version='12.1.2.1' />
```
Comment (cero-miedo) [inline product_docs/docs/ocl_connector/12.1.2.1/index.mdx:16]: this needs to be removed

File: product_docs/docs/ocl_connector/12.1.2.1/index.mdx [hunk 2]
```diff
@@ -1,10 +1,10 @@
 ---
 title: EDB OCL Connector
-productStub: true
+#productStub: true
```
Comment (cero-miedo) [inline product_docs/docs/ocl_connector/12.1.2.1/index.mdx:3]: this needs to be removed too

File: product_docs/docs/ocl_connector/12.1.2.1/01_whats_new.mdx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+title: "What’s New"
+
+legacyRedirectsGenerated:
```
Comment (cero-miedo) [inline product_docs/docs/ocl_connector/12.1.2.1/01_whats_new.mdx:4]: these redirect either need to be removed, or changed to reference 12.1.12.1 for each file

**Review by cero-miedo — CHANGES_REQUESTED** [review]
changes requested on `index.mdx` and all redirects need to be explicit to the version, or removed.

**Comment by ManjushaVaidya** [comment]
Made the required changes, please check and merge

File: product_docs/docs/ocl_connector/12.1.2.1/index.mdx [hunk 4]
```diff
@@ -1,10 +1,9 @@
 ---
 title: EDB OCL Connector
-productStub: true
 directoryDefaults:
   description: "EDB OCL Connector Version 12.1.2.1 Documentation and release notes."
 
-legacyRedirectsGenerated:
+#legacyRedirectsGenerated:
```
Comment (cero-miedo) [inline product_docs/docs/ocl_connector/12.1.2.1/index.mdx:6]: this is breaking the build, if the intent is to redirect content, it needs to be uncommented. If it needs to be removed, the indented lines below also need to be removed

**Review by cero-miedo — CHANGES_REQUESTED** [review]

File: product_docs/docs/ocl_connector/12.1.2.1/index.mdx (hunk 4 above)
Comment (ManjushaVaidya) [reply product_docs/docs/ocl_connector/12.1.2.1/index.mdx]: I have removed it, please check

**Pull request closed without merging**